myapp/db/schema.rb

```diff
@@ -10,7 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2024_05_24_044922) do
+ActiveRecord::Schema.define(version: 2024_05_27_043131) do
+
+  create_table "labels", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci", force: :cascade do |t|
```

Comment:
This is for different step?

Reply:
yes... sorry I don't know why this change is included in this PR.
Maybe I did something wrong when I switched branches...